packages/cli/src/index.ts

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+const { program } = require('commander')
+const { exec } = require('child_process');
+const { version: VERSION } = require('../package.json');
+
+
+program
+    .command('version')
+    .description('output the version number')
+    .action(version)
```

Comment:
does this work or should this use the alias `VERSION` ?

Reply:
this works, I'm using a function to log the VERSION